cmd/run.go

```
@@ -101,7 +108,7 @@ func runTask(taskRequest TaskRequest) error {
 	// Check that experimental mode is enabled on the daemon, fall back to no logging if not
 	versionInfo, versionErr := c.ServerVersion(context.Background())
 	if versionErr != nil {
-		log.Fatal("Is the Docker Daemon running?")
+		log.Fatal("Error " + versionErr.Error() + ". Is the Docker Daemon running?")
```

could use [`log.Fatalf`](https://golang.org/pkg/log/#Fatalf) instead.

---

though I'd rather return the error actually.